include/mbedtls/ccm.h

```diff
@@ -55,6 +55,11 @@
 
 #include "mbedtls/cipher.h"
 
+#define MBEDTLS_CCM_DECRYPT       0
+#define MBEDTLS_CCM_ENCRYPT       1
+#define MBEDTLS_CCM_STAR_DECRYPT  2
```

Comment:
This is to be able to allow in mbedtls_ccm_finish() a tag length of 0 for CCM* but not for CCM as one-shot functions do.

Reply:
Ok. I don't think this is necessary for a low-level API, but it makes sense to do the same as the one-shot functions. (They handle CCM* a bit differently because it was added after the fact.)